### ISSUE TITLE
[Feature Request]: System Locations items must also be customizable during installation.

### ISSUE THREAD

**Issue by Neytiri7** [Comfy-Org/desktop#399]
### Prerequisites

- [X] I have searched the existing issues and checked the recent builds/commits
- [X] I have read the documentation and this feature is not already available
- [X] This is a single feature request (not multiple features combined)

### Problem Statement

During installation, you can specify the path to install the ComplyUI Desktop in Item 1,
The System Locations path is fixed.

### Proposed Solution

During installation, the System Locations path must also be changeable by the user.

### Potential workflow

1. Run the installation exe file
2. Choose Installation Location 항목 확인
3. Verify that System Locations entries at the bottom of the screen can be changed

### Impact Level

High (Major improvement)

### Additional information

![Screenshot 2024-11-30 at 05 04 10](https://github.com/user-attachments/assets/75f44d86-0760-4e49-ba85-5d860da3dbcb)

**Comment by rainlizard** [comment]
Yeah I'm so confused by how I've got two comfyui folders now.

- C:\AI\ComfyUI
- C:\Users\***\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI

They've both got directories for models/loras/embeddings, etc.

When I installed custom nodes they went into the 2nd one. But I put all my models, loras, embeddings into the 1st folder. So now my files are all split up. Why is it like this?

**Comment by NEMStudios** [comment]
Yes, I tried to transfer it to a different disk directory using the link folder way, such as the I drive, and everything did seem to work out, and there were no obvious problems with the plugin installation loading, but once I started executing the node persona it couldn't be done. It should still be a problem with System Location,I think all the paths installed through the desktop package、Execution configuration files, etc.,Should be customized by the user,Since there is this path option during installation,Then you should be able to choose the option of all the file paths that will be installed when you install,You should not bind System Location in the C drive,You must know that it is too heavy to put all the pressure on the C disk
